src/components/EditorPage.js

```diff
@@ -108,6 +138,29 @@ class EditorPage extends Component {
             <i className='ion-android-arrow-back' style={{ marginRight: 0 }} />
           </Button>
 
+          <span className={cx('EditorPage-bar-item', {
+            editable: true,
+            hover: mouseOverName,
+            edit: editName,
```

Comment:
No need to add the `.hover` class, it's already handled by `:hover` in CSS, and no need to add the `.editable` class as well (no associated CSS 😄 ). Finally, the `.edit` class is never added because `editName` is never set in state. Si basically you can just have
```
<span className='EditorPage-bar-item'>
```